ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoInputHelper.java

```diff
@@ -35,89 +29,4 @@ public static void synchronize(Crac crac, Network network) {
             LOGGER.debug("Crac {} is already synchronized", crac.getId());
         }
     }
-
-    public static List<String> cleanCrac(Crac crac, Network network) {
```

Comment:
Thanks for the migration :)

Though, if you do that,  you will also have to change the dependencies of farao-core (cucumber, core-rte and camarero).

If you prefer you can keep this method here. We can do it in two steps : 

- first you create the new package (without caring about duplication) which covers what you need, and you plug it on the flow-based tool, 

- then in another PR ( that we could handle if you do not want to stir the RAO) we migrate the cleanCrac of the RAO to your new package, making sure we don't have regressions

Reply:
I'll restore this deleted method tomorrow, and put it as deprecated. It won't be used anymore here and in core-rte, but I won't check in camarero and cucumber.

Reply:
Restored!

Reply:
Could you revert it completely (avoid the renaming, which is not needed)

Reply:
Better solution, as discussed